components/readme.adoc

```diff
@@ -936,6 +936,8 @@ Number of Data Formats: 43 in 35 JAR artifacts (0 deprecated)
 
 | link:camel-flatpack/src/main/docs/flatpack-dataformat.adoc[Flatpack] (camel-flatpack) | 2.1 | The Flatpack data format is used for working with flat payloads (such as CSV, delimited, or fixed length formats).
 
+| link:camel-grok/src/main/docs/grok-dataformat.adoc[Grok] (camel-grok) | 3.0 | To use a grok data format.
```

Comment:
This adoc file seems to be absent. It would be good to add one.

Reply:
True, adoc and javadocs added in e707a61d6051be9b8f84ddebaac2b6975934eefd